oioioi/base/templatetags/format_data_range.py

```diff
@@ -1,11 +1,15 @@
+from django.utils.timezone import template_localtime
 from django import template
 from django.utils import formats
 
 register = template.Library()
 
-@register.simple_tag
-def format_data_range(start, end):
+@register.simple_tag(takes_context=True)
+def format_data_range(context, start, end):
     if start is not None and end is not None:
+        start = template_localtime(start, context.use_tz)
```

Comment:
if start is not None and end is not None, then both start/end are passed through template_localtime 2 times (in line 10 & in line 21),
I suggest to move initialisation to the beginning of the function, like:
start = template_localtime(start, context.use_tz) is start is not None else None